Source/WebCore/rendering/BidiRun.h

```diff
@@ -33,6 +34,8 @@ class LegacyInlineBox;
 class RenderObject;
 
 struct BidiRun : BidiCharacterRun {
+    WTF_MAKE_STRUCT_TZONE_ALLOCATED(BidiRun);
+public:
```

Comment:
I think this is also not needed right? Since `WTF_MAKE_STRUCT_TZONE_ALLOCATED` has the definition:
```
#define MAKE_STRUCT_BTZONE_MALLOCED(isoType, heapType, exportMacro) \
    MAKE_BTZONE_MALLOCED_COMMON(isoType, heapType, exportMacro) \
    using __makeTZoneMallocedMacroSemicolonifier BUNUSED_TYPE_ALIAS = int
```